src/Lumper.Lib/BSP/Lumps/Lump.cs

```diff
@@ -3,30 +3,29 @@ namespace Lumper.Lib.BSP.Lumps;
 using JsonSubTypes;
 using Newtonsoft.Json;
 
-public abstract class Lump<T> : Lump
-    where T : System.Enum
 // Lump with an enum type variable
+public abstract class Lump<T>(BspFile parent) : Lump(parent) where T : System.Enum
 {
-    public T Type { get; set; }
-    protected Lump(BspFile parent) : base(parent)
-    { }
+    public T Type { get; set; } = default!;
 }
 
 // Implements how a lump will be read from a stream (same for write).
 // Doesn't store offset or length because header information is separate.
 [JsonConverter(typeof(JsonSubtypes), "Class")]
-public abstract class Lump
+public abstract class Lump(BspFile parent)
```

Comment:
Okay, I'll do some of these this afternoon. In this case it actually simplifies the subclasses of `Lump` a lot, since they no longer need to pass `parent` to the base class.

Note that there's quite a few cases where object initializers are not applicable, usually when a class has multiple constructors, or where significant work is being done in the constructor.